File: tasks/2024/round2_pp/kacs.py
#!/usr/bin/env python3
# @check-accepted: examples Teljes
# insert brief description of the solution here


# input data

# insert your code here
N =  int(input())

T = list(map(int, input().split()))
ans=0
for t in T:
 	if t >= 1982:
    		ans+=1
print(ans)

File: tasks/2024/round2_pp/kacs.cpp
// @check-accepted: examples Teljes
// insert brief description of the solution here

#include <fstream>
#include <iostream>
#include <vector>

using namespace std;

int main() {
    // uncomment the following lines if you want to read/write from files
    // ifstream cin("input.txt");
    // ofstream cout("output.txt");
    int N, ans = 0, t;

    cin >> N;
    for (int i = 0; i < N; i++) {
        cin >> t;
        if (1982 <= t)
            ans++;
    }
    cout << ans << "\n";

    return 0;
}
